packages/approval-controller/src/ApprovalController.test.ts

```diff
@@ -826,6 +832,15 @@ describe('approval controller', () => {
 
       const result = await approvalPromise;
       expect(result).toBe('success');
+
+      expect(acceptedEvent.mock.calls[0][0]).toEqual({
```

Comment:
For maintainability and modularity, should we have distinct tests to verify the events are fired?